raiden_contracts/tests/test_channel_close.py

```diff
@@ -301,6 +309,37 @@ def test_close_first_participant_can_close(
         EMPTY_SIGNATURE,
     ).transact({'from': A})
 
+    (
+        settle_block_number,
+        state,
+    ) = token_network.functions.getChannelInfo(channel_identifier, B, A).call()
+    assert settle_block_number == settle_timeout + get_block(close_tx)
+    assert state == ChannelState.CLOSED
+
+    (
+        _, _,
+        A_is_the_closer,
+        A_balance_hash,
+        A_nonce,
+        _,
+        _,
+    ) = token_network.functions.getChannelParticipantInfo(channel_identifier, A, B).call()
+    assert A_is_the_closer is True
```

Comment:
Some people would say that `... is True` is syntactic noise and you should just `assert A_is_the_closer`. But in this case it's not totally obvious from the context that `A_is_the_closer` is a boolean, so this does make it a tiny bit clearer.

Reply:
I looked around. `... is True` is consistently used in this file, so I follow the local custom for now.